res/TensorFlowLiteRecipes/Net_StridedSlice_StridedSlice_000/test.recipe

```diff
@@ -0,0 +1,92 @@
+operand {
+  name: "ifm"
+  type: INT32
+  shape { dim: 1 dim: 2 dim: 4 dim: 4 }
+}
+operand {
+  name: "begin"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "1" arg: "1" arg: "1" arg: "1"
```

Comment:
I think this would be 
```
    arg: "0" arg: "0" arg: "0" arg: "0"
```
to make output shape same with input.

Reply:
Done